lib/feature/profile/account_detail/cubit/account_detail_cubit.dart

```diff
@@ -84,6 +84,7 @@ class AccountDetailCubit extends Cubit<AccountDetailState> {
               .map((e) => e.address)
               .contains(address)) {
         _subCreatedManually = true;
+        // ignore: prefer-async-await
```

Comment:
Why do you prefer `then` rather than make this method async and `await` for futures?

Reply:
-